### PR TITLE
Creating DataAnalytics Repo

### PR DESCRIPTION
**Pull request by kruti-m**
Created a New File Directory content/DataAnalytics
 new file:   content/DataAnalytics/introduction.md
 modified:   content/DataAnalytics/introduction.md

### CODE REVIEW

**Comment by kruti-m** [comment]
Created a New File **Directory content/DataAnalytics**
new file: content/DataAnalytics/introduction.md,
content/DataAnalytics/typeofdata.md
content/DataAnalytics/dataoperations.md
content/DataAnalytics/splitofdata1.md
content/DataAnalytics/splitofdata2.md
content/DataAnalytics/splitofdata3.md
content/DataAnalytics/serviceselection.md
content/DataAnalytics/QnA.md

Images Added
 images/DataWarehouse.png
 images/OLTP.png 
 images/DataHeirarchyOLTPtoOLAP.png 
 images/DataLake.png 
 images/DataLaketoLakehouse.png 
 images/comparing-data-lake-and-data-warehouse.png 
 images/DeltaTables.png 
 images/DeltaLakeQueries.png 
 images/DeltaLakeUsage.png 
 images/ConsolidationIcon.png 
 images/ExtractionIcon.png 
 images/BatchProcessing.png 
 images/StreamProcessing.gif
 images/DeltaLakeTopofLakehouse.png 
 images/MicrosoftPurview.png 
 images/DataAnalyticsFlowChart.png 
 images/DataAnalyticsHeirarchy.png

**Review by BennyHarding — COMMENTED** [review]
Formatting for the learning paths is broken.
Any reason to include Fabric and Analytics learning paths for this file?
What was wrong with the existing learning path links?

File: content/data/sqlmigration/QnA.md
Comment (BennyHarding) [inline content/data/sqlmigration/QnA.md:1]: Formatting for the learning paths is broken.
Any reason to include Fabric and Analytics learning paths for this file?
What was wrong with the existing learning path links?

File: content/data/sqlmigration/discoveryandassessment.md
Comment (BennyHarding) [inline content/data/sqlmigration/discoveryandassessment.md:1]: All good

File: content/data/sqlmigration/QnA.md
Comment (kruti-m) [reply content/data/sqlmigration/QnA.md:1]: Not sure
I updated it yesterday with the new collections I created 
They are specific to each section

**Comment by kruti-m** [comment]
In creating the DataAnalytics Repo noticed some minor changes in the sqlmigration (broken links and additional links.) Appended the same

**Review by BennyHarding — APPROVED** [review]
New content for intro in Data & Analytics topics

**Review by BennyHarding — APPROVED** [review]

**Comment by kruti-m** [comment]
Updating DataAnalytics -> End to End pipeline branch for Main

**Pull request merged**